src/frontend/components/SøknadsSteg/Utvidet-DinLivssituasjon/useDinLivssituasjon.tsx

```diff
@@ -170,7 +171,7 @@ export const useDinLivssituasjon = (): {
         },
         ESvar.JA,
         harSamboerNå,
-        true
+        felt => validerDato(felt, true)
```

Comment:
Er dette endra for å gjøre det mer leselig? I så fall, hva med en liten utility-funksjon med signatur `avgrensDatoFremITid(felt): boolean` slik at man her bare passer inn den? For meg ville i alle fall det vært enklere enn å forstå siden jeg ikke her vet hva validerDato gjør og måtte gå til intellij for å finne ut av det.

Reply:
Hovedgrunnen er at useOmBarnet linje 515 trenger en annen valideringsfunksjon enn useDatovelgerFeltMedJaNeiAvhengighet.ts  hadde fra før. Men kan fikse litt på looken der som du nevner